### PR TITLE
add recipe for prometheus-memcached-exporter

### PR DESCRIPTION
**Pull request by Guidurand**

### CODE REVIEW

File: p/prometheus-memcached-exporter/files/init [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    set "$0" "$@"; INIT_D_SCRIPT_SOURCED=true . /lib/init/init-d-script
+fi
+### BEGIN INIT INFO
+# Provides:          prometheus-memcached-exporter
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Prometheus exporter for machine metrics
+# Description:       Prometheus exporter for machine metrics
```
Comment (vbatoufflet) [inline p/prometheus-memcached-exporter/files/init]: Could you please fix both `Short-Description` and `Description` fields to a different description than `prometheus-node-exporter` one?

File: p/prometheus-memcached-exporter/files/init [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+### END INIT INFO
+
+NAME="prometheus-memcached-exporter"
+DESC="Prometheus exporter for machine metrics"
```
Comment (vbatoufflet) [inline p/prometheus-memcached-exporter/files/init]: Could you also please fix this one?

File: p/prometheus-memcached-exporter/recipe.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+version: 1
+
+name: prometheus-memcached-exporter
+description: Prometheus exporter for machine metrics
```
Comment (vbatoufflet) [inline p/prometheus-memcached-exporter/recipe.yaml]: And this one 😉

Comment (Guidurand) [reply p/prometheus-memcached-exporter/recipe.yaml]: fixed in commit 61736f8686a6c05857894a4ebd34ccecf8465b6b

**Pull request merged**